integration/src/test/java/org/mule/test/IntegrationTestCaseRunnerConfig.java

```diff
@@ -9,7 +9,8 @@
 
 import org.mule.test.runner.ArtifactClassLoaderRunnerConfig;
 
-@ArtifactClassLoaderRunnerConfig(sharedRuntimeLibs = {"org.mule.tests:mule-tests-unit", "org.mule.tests:mule-tests-functional"})
+@ArtifactClassLoaderRunnerConfig(sharedRuntimeLibs = {"org.mule.tests:mule-tests-unit", "org.mule.tests:mule-tests-functional",
+    "org.mule.tests:mule-tests-component-plugin"})
```

Comment:
This seems wrong to me. Why the app has to share code that is provided by the plugin?